services/user/Homepage/ui/src/components/nav-user.tsx

```diff
@@ -111,14 +162,7 @@ export function NavUser() {
                             >
                                 <Avatar className="h-8 w-8 rounded-lg">
                                     <AvatarImage
-                                        src={
-                                            chainId && currentUser
-                                                ? generateAvatar(
-                                                      chainId,
-                                                      currentUser,
-                                                  )
-                                                : undefined
-                                        }
+                                        src={avatarSrc}
```

Comment:
An image that's not perfectly square gets stretched in the nav user menu. We should probably use `object-fit: cover;` here and below.

Reply:
Other places where this component is used should also `cover` (contacts list, contact page, etc.)

Reply:
Profile image is still stretched on the contact page:
<img width="562" alt="image" src="https://github.com/user-attachments/assets/62793455-6a26-434c-a0c8-e710eaa204b2" />